applications/StructuralMechanicsApplication/custom_utilities/formfinding_IO_utility.cpp

```diff
@@ -0,0 +1,94 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher 
+//
+	        
+
+// System includes
+
+
+// External includes 
+
+
+// Project includes
+#include "includes/checks.h"
+#include "formfinding_IO_utility.h"
+#include "structural_mechanics_application_variables.h"
+#include "includes/model_part_io.h"
+#include "includes/model_part.h"
+
+
+namespace Kratos
+{
+
+	FormfindingIOUtility::FormfindingIOUtility(ModelPart& rModelPart, const Parameters rParameter): 
+		mModelPart(rModelPart)
+		{}
+    void FormfindingIOUtility::PrintModelPart(){
+		// erase nodal data
+    	mModelPart.GetNodalSolutionStepVariablesList().clear();
+
+    	// erase elemental data
+    	for( auto& ele: mModelPart.Elements()){
+    	    const Variable<Matrix> variable = KratosComponents<Variable<Matrix>>::Get("MEMBRANE_PRESTRESS");
+    	    if(ele.Has(variable)){
+    	        const Matrix membrane_prestress(ele.GetValue(MEMBRANE_PRESTRESS));
+    	        ele.Data().Clear();
+    	        ele.SetValue(MEMBRANE_PRESTRESS, membrane_prestress);
+    	    }
+    	    else
+    	        ele.Data().Clear();
+
+    	}
+
+    	// erase conditional data
+    	for( auto& cond: mModelPart.Conditions()){
+    	        cond.Data().Clear();
+    	}
```

Comment:
you can remove the brackets for these loops

Reply:
If you are happy afterwards I am glad to do it :)